Source/Core/Core/MemoryWatcher.cpp

```diff
@@ -62,12 +77,26 @@ void MemoryWatcher::ParseLine(const std::string& line)
 
 bool MemoryWatcher::OpenSocket(const std::string& path)
 {
+#ifdef __unix__
 	memset(&m_addr, 0, sizeof(m_addr));
 	m_addr.sun_family = AF_UNIX;
 	strncpy(m_addr.sun_path, path.c_str(), sizeof(m_addr.sun_path) - 1);
 
 	m_fd = socket(AF_UNIX, SOCK_DGRAM, 0);
 	return m_fd >= 0;
+#elif defined(_WIN32)
+	pipe = CreateNamedPipe(L"\\\\.\\dolphin-emu\\mem-watch", 
+		PIPE_ACCESS_OUTBOUND,
+		PIPE_TYPE_BYTE | PIPE_WAIT,
+		1,
+		0,
+		0,
+		0,
+		NULL);
+
+	if(pipe == NULL | pipe == INVALID_HANDLE_VALUE) return false;
```

Comment:
Use || for booleans. | is a different operator.

And [docs](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365150%28v=vs.85%29.aspx) say NULL is not the error return, so no point checking that at all.